### DISCUSSION TITLE
Welcome to ScalaFX Discussions!

### DISCUSSION THREAD

**Discussion by jpsacha** [scalafx/scalafx#330] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Benevolent** [comment]
Hi, I'm a casual developer, my Jeckyll & Hide manifestation as I'm more busy with other activities for the moment at least. Not regretting being stuck with scalafx as of my desktop project.Recently switched to Linux platform and things are a bit smoother than in Windows for me in terms of development. I'm in favor of sharing good code, and contributing, but would like that to be of adequate quality as well. Of the last part I am rather hesitant.
Even if it is rather quiet, I look forward having valuable exchange with other casual and non coders.
